### PR TITLE
remove new line from logs of sendUpdateValueNgsi2

### PR DESCRIPTION
**Pull request by AlvaroVega**
that characters where introduced by https://github.com/telefonicaid/iotagent-node-lib/issues/1342

### CODE REVIEW

**Review by mapedraza — APPROVED** [review]
LGTM

File: lib/services/ngsi/entities-NGSI-v2.js [hunk 1]
```diff
@@ -347,7 +347,7 @@ function isFloat(value) {
 function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, callback) {
```
Comment (fgalan) [inline lib/services/ngsi/entities-NGSI-v2.js:347]: Maybe CNR should include something like this?

```
- Fix: avoid usage of newlines in logs
```

Comment (AlvaroVega) [reply lib/services/ngsi/entities-NGSI-v2.js:347]: Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1356/commits/0d81152d8cc0aa80f21c2ec2de2112ae60e1b83d

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,5 +1,6 @@
+- Fix: avoid usage of newlines in logs
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: avoid usage of newlines in logs (#1342)

```

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -1,5 +1,7 @@
+- Fix: avoid usage of newlines in logs (#1342)
+
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:2]:
```suggestion

```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**